interpreter/convert.go

```diff
@@ -18,39 +24,44 @@ func convertToStarlark(value interface{}) (starlark.Value, error) {
 	case uint64:
 		return starlark.MakeUint64(v), nil
 	case float32:
+		return starlark.Float(v), nil
```

Comment:
do we need this or will it just do fall through?

Reply:
Also, maybe we should just transition over to just using reflect so there's only one switch?

Reply:
it doesn't fall through